spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -378,6 +382,27 @@ class CometSparkSessionExtensions
             case None => b
           }
 
+        case s @ ShuffleQueryStageExec(_, _: CometShuffleExchangeExec, _) =>
```

Comment:
I'd suggest highlighting the difference between these 2 blocks with comments.  The actions are exactly the same, so the only difference is the case check.

Reply:
Okay